### ISSUE TITLE
Android App Running on Device

### ISSUE THREAD

**Issue by ashar02** [sign/translate#152]
### Current Behavior

When I compile and run the android app on device then tap on audio record button. Following error appears on console:
E/Capacitor/Console: File: https://localhost/src_app_pages_translate_translate_module_ts.8d0325fdc8c8b657.js - Line 1 - Msg: error not-allowed

Although I can see in settings that app has audio record permission. Web app on android browser is working perfectly in same case. Any hint regarding this issue? 

### Example flow

```
paste your flow here
```

### Environment

- Device: (_eg._ iPhone Xs; laptop)
- OS: (_eg._ iOS 13.5; Fedora 32)
- Browser: (_eg._ Safari; Firefox)
- Version: (_eg._ 13; 73)
- Other info: (_eg._ display resolution, ease-of-access settings)

**Comment by AmitMY** [comment]
- did it prompt you for audio permissions, and you allowed them?
- it might be that in the manifest we don't correctly declare audio permissions

**Comment by ashar02** [comment]
Yes, app is asking for permission and settings app showing that app has permission too. Device is android 13. If app is asking permission it means manifest has proper declaration. I am further investigating it.

**Comment by ashar02** [comment]
E/Capacitor/Console: File: https://localhost/main.7905458a6f5a6d3b.js - Line 1 - Msg: ERROR Error: not implemented
E/Capacitor/Console: File: https://localhost/main.7905458a6f5a6d3b.js - Line 1 - Msg: Service worker registration failed with: TypeError: Failed to register a ServiceWorker for scope ('https://localhost/') with script ('https://localhost/ngsw-worker.js'): An unknown error occurred when fetching the script.
E/Capacitor/Console: File: https://localhost/src_app_pages_translate_translate_module_ts.c64fa9e78a070475.js - Line 1 - Msg: error not-allowed
E/Capacitor/Console: File: https://localhost/main.7905458a6f5a6d3b.js - Line 1 - Msg: ERROR [object DOMException]

**Comment by AmitMY** [comment]
Thanks for investigating, I'm not sure I can assist on this.
There should be support for the `SpeechRecognition` web API.

![image](https://github.com/sign/translate/assets/5757359/78baa796-a4f2-47bc-9afb-569ce15cfff3)

**Comment by ashar02** [comment]
Thanks for sharing above info.
